ui/shared/components/page/WarningMessages.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Grid, Message } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+
+import { getWarningMessages } from 'redux/selectors'
+
+
+class WarningMessages extends React.PureComponent {
+
+  static propTypes = {
+    warningMessages: PropTypes.array,
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {}
+  }
+
+  hide = message => () => {
+    this.setState({ [message]: true })
+  }
+
+  render() {
+    const warningMessages = this.props.warningMessages.filter(({ message }) => !this.state[message])
+    return warningMessages && warningMessages.length > 0 && warningMessages.map(({ header, message }) =>
```

Comment:
The first `warningMessages` in the return statement may not be necessary. Instead, `this.props.warningMessages` could be undefined if the backend didn't provide the `warningMessages` meta field.